spec.md

```diff
@@ -1027,12 +1027,20 @@ fail.
 A dictionary used in a Boolean context is considered true if it is
 non-empty.
 
-The binary `+` operation may be applied to two dictionaries.  It
-yields a new dictionary whose elements are the union of the two
-operands.  If a key is present in both operands, the result contains
-the value from the right operand.
-<b>Note:</b> this feature is deprecated.  Use the
-`dict.update` method instead.
+The binary `|` operation may be applied to two dictionaries. It yields a new
+dictionary whose set of keys is the union of the sets of keys of the two
+operands. The corresponding values are taken from the operands, where the value
+taken from the right operand takes precedence if both contain a given key.
+Iterating over the keys in the resulting dictionary first yields all keys in
+the left operand in insertion order, then all keys in the right operand that
+were not present in the left operand, again in insertion order.
+<b>Note:</b> Previously, the binary `+` operator was used for dictionary unions.
+This is now deprecated.
```

Comment:
Remove the "note": it was not part of the spec

Reply:
Dropped it.